bitcoin/src/merkle_tree/block.rs

```diff
@@ -514,14 +514,18 @@ impl std::error::Error for MerkleBlockError {
 
 #[cfg(test)]
 mod tests {
-    use hex::test_hex_unwrap as hex;
-    #[cfg(feature = "rand-std")]
-    use secp256k1::rand::prelude::*;
```

Comment:
So `prelude` modules are "special" in Rust, they are supposed to hold stuff that every user of a lib wants and are expected to be imported with a wildcard like this (@apoelstra does have an aversion in general to wildcard imports though).

Reply:
FTR I have an aversion to wildcard imports too. ;)